compiler/test/compilable/test22727.c

```diff
@@ -3,9 +3,10 @@
 int fooc(int a) { return a; }
 
 __stdcall int foostdcall(int a) { return a; }
+_stdcall int _foostdcall(int a) { return a; } // test issue 24509
```

Comment:
this should be called `_foostdcall3`, it will cause a redefinition error otherwise.

Reply:
done